capstone/src/main/java/com/step902020/capstone/ReviewController.java

```diff
@@ -0,0 +1,83 @@
+package com.step902020.capstone;
+
+import com.step902020.capstone.security.CurrentUser;
+import java.io.IOException;
+import java.util.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.view.RedirectView;
+import com.google.gson.Gson;
+import java.io.IOException;
+
+
+/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+@RestController
+public class ReviewController {
+
+  @Autowired
+  private ReviewRepository reviewRepository;
+
+  @Autowired
+  private IndividualRepository individualRepository;
+
+  /**
+   * Toggle user's like. Add like if they have not already liked it. Remove like if they have already liked it
+   * @param user current user
+   * @param reviewId review's datastore id
+   * @return amount of review's likes
+   */
+  @PostMapping("review-likes")
+  public int reviewLikes(
```

Comment:
Consider naming the method with an action, such as `updateReviewLikes()`.
The URL could go either way (stay as is or change to action). The pure REST convention is to have a noun as URL, and use the method (such as POST) to convey action. But you see plenty of action verbs in URLs, too, because they are easy for people to immediately understand.